bootstore/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+//! The two communication paths for the bootstore:
```

Comment:
Nit: Should add the MPL header to this file

Reply:
Done.